app/src/main/java/com/example/mykotlincalculator21/mvp/contract/CalculatorContract.kt

```diff
@@ -1,5 +1,7 @@
 package com.example.mykotlincalculator21.mvp.contract
 
+import com.example.mykotlincalculator21.utils.ErrorUtils
+
 interface CalculatorContract {
```

Comment:
please do not abuse with the empty lines

Reply:
Thx for the tip, now they are no more